src/main/scala/linting/LintAnonymousRegisters.scala

```diff
@@ -0,0 +1,134 @@
+package freechips.rocketchip.linting
+
+import firrtl._
+import firrtl.ir._
+import firrtl.traversals.Foreachers._
+import firrtl.annotations.NoTargetAnnotation
+import firrtl.options.{OptionsException, RegisteredLibrary, ShellOption}
+import firrtl.stage.RunFirrtlTransformAnnotation
+import chisel3.experimental.ChiselAnnotation
+
+
+import scala.collection.mutable
+
+case class LintError(info: FileInfo, modules: Set[String])
+case class LintExceptions(seq: Seq[LintError]) extends FirrtlUserException(
```

Comment:
Thanks for making the other names more specific, but these two are still pretty generic, what is the plan for adding other lint errors?